protocols/v2/binary-sv2/no-serde-sv2/codec/src/codec/decodable.rs

```diff
@@ -51,7 +78,10 @@ pub trait Decodable<'a>: Sized {
     }
 }
 
-/// Passed to a decoder to define the structure of the data to be decoded
+// Enum representing primitive data markers.
+//
+// These markers are used to identify primitive types such as integers, booleans, and byte arrays.
+// Each variant represents a specific type and is used during decoding to interpret raw data correctly.
 #[derive(Debug, Clone, Copy)]
 pub enum PrimitiveMarker {
```

Comment:
The reason why I didn't did this was because of that top level encapsulation what we saw, in lib.rs. So, because of that this wont be rendered and will not be visible to users of this crate.